bin/p2-log-bridge/main.go

```diff
@@ -1,10 +1,11 @@
 package main
 
 import (
-	"io"
+	"fmt"
 	"os"
 	"os/exec"
-	"os/signal"
+	"sync"
+	"syscall"
```

Comment:
Should new code use [`golang.org/x/sys/unix`](https://godoc.org/golang.org/x/sys/unix)? I don't think it changes much in the code below?